MAINTAINERS.md

```diff
@@ -21,15 +21,15 @@ This document contains a list of maintainers in this repo. See [opensearch-proje
 | Sean Kao          | [seankao-az](https://github.com/seankao-az)         | Amazon      |
 | Max Ksyunz        | [MaxKsyunz](https://github.com/MaxKsyunz)           | BitQuill    |
 | Yury Fridlyand    | [Yury-Fridlyand](https://github.com/Yury-Fridlyand) | BitQuill    |
+| Anirudha Jadhav   | [anirudha](https://github.com/anirudha)             | Amazon      |
 
 ## Emeritus Maintainers
 
 | Maintainer        | GitHub ID                                               | Affiliation |
 | ----------------- | ------------------------------------------------------- | ----------- |
 | Charlotte Henkle  | [CEHENKLE](https://github.com/CEHENKLE)                 | Amazon      |
-| Anirudha Jadhav   | [anirudha](https://github.com/anirudha)                 | Amazon      |
```

Comment:
Why move?

Reply:
Apparently Emeritus means he will not have write access/approve access so we are moving him back. The Codeowners file should go green once @bbarani updates the repo permissions